testing/file_test/run_test.cpp

```diff
@@ -122,7 +122,7 @@ auto RunTestFile(const FileTestBase& test_base, bool dump_output,
       input_stream = tmpfile();
       fwrite(split->content.c_str(), sizeof(char), split->content.size(),
              input_stream);
-      rewind(input_stream);
+      CARBON_CHECK(!fseek(input_stream, 0, SEEK_SET));
```

Comment:
```suggestion
      CARBON_CHECK(fseek(input_stream, 0, SEEK_SET) == 0);
```

I find the comparison to be more clear in cases like this where 0 is the success, and it's not a boolean.